modules/rodoo-uninstall-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/run_once_duration_override/run-once-duration-override-uninstall.adoc
+
+:_content-type: PROCEDURE
+[id="rodoo-uninstall-operator_{context}"]
+= Uninstalling the {run-once-operator}
+
+You can use the web console to uninstall the {run-once-operator}.
+
+// TODO: What happens to pods with that value set after RODOO is uninstalled? Do they keep that value, but will lose it if updated (and any new ones will no longer get it)?
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* You have installed the {run-once-operator}.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. Navigate to *Operators* -> *Installed Operators*.
+. Select `openshift-run-once-duration-override-operator` from the *Project* dropdown list.
+. Click the Options menu {kebab} next to the *{run-once-operator}* entry and click *Uninstall Operator*.
++
```

Comment:
@bergerhoffer we should add steps to remove RODO CR object before uninstalling the operator completely like how we do for descheduler